src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -339,7 +341,7 @@ private void RefreshWhenNotExecutingAction(RefreshOperationSource source, TimeSp
             }
         }
 
-        private void EmitRefreshEvent(TimeSpan timeSpan, RefreshOperationSource refreshOperationSource, RefreshOperationStatus status)
+        private void EmitRefreshEvent(TimeSpan timeSpan, RefreshOperationSource refreshOperationSource, RefreshOperationStatus status, bool isUIFiltering = false)
```

Comment:
once again...is this filter from the search query? or filtering based on updates available?  

i'd recommend using a clarfying term with Filtering...or don't use filtering term at all.

Reply:
I did use a clarifying term, by saying "UI filtering". Can you suggest something more clear?

Reply:
You are filtering the UI.
But are you filtering it with the search term in the search box?
Or are you filtering the items for those that have updates?
etc...

what are you really doing here? what are some other terms that you could describe that variable as?

isInstalledOrUpdatesTab?
didnotHaveToGetDataCauseTheDataWasAlreadyThere? (very verbose form...just asking to understand what this should be called)
etc...

---
In reply to: [450528041](https://github.com/NuGet/NuGet.Client/pull/3444#discussion_r450528041) [](ancestors = 450528041)

Reply:
This PR is just filtering the Updates- vs No-Updates- Available. Sure, at some point, it may make sense to make search terms smart and use UI filtering as well.

isInstalledOrUpdatesTab - invalid as soon as we want to apply UI filter other tabs/scenarios

What else to call it? Not sure, but WPF calls it a Filter, and it happens at the UI level. I don't really see confusion with the verbiage "UI Filtering", and certainly not more confusion than the proposals bring with them.